gateway/src/test/resources/configuration/gateway.default.yaml

```diff
@@ -118,3 +118,16 @@
 # Enables long polling for available jobs
 # This setting can also be overridden using the environment variable ZEEBE_GATEWAY_LONGPOLLING_ENABLED.
 # enabled: true
+
+# multiTenancy:
+# Enables multi tenancy for the gateway.
+# When enabled, the gateway enhances requests with the authorized tenant ids of the requester.
+# It also rejects requests that specify a tenant id that the requester is not authorized to access.
+# When disabled, the gateway enhances requests with the default tenant id.
```

Comment:
:wrench: Similar to the above, it might be good that we document the value of the default tenant id here, so that future Zeebe devs are more aware of it.

```suggestion
# When disabled, the gateway enhances requests with the <default> tenant id.
```